dev/docker/cudf/Dockerfile

```diff
@@ -1,9 +1,7 @@
 FROM ghcr.io/facebookincubator/velox-dev:adapters
 RUN yum install -y sudo patch maven perl
-# After the base docker image updated, don't need to install cmake
-RUN pip install cmake==3.30.4 && ln -sf /usr/local/bin/cmake /usr/bin
 
-RUN git clone -b cudf --depth=1 https://github.com/apache/incubator-gluten /opt/gluten
+RUN git clone --depth=1 https://github.com/apache/incubator-gluten /opt/gluten
```

Comment:
Can we still uses the branch cudf? I have updated it to use the rapidsai velox to enable all gpu feature